### PR TITLE
Fix list style based on comment from @reschke

### PR DESCRIPTION
**Pull request by yoavweiss**
@reschke - here's the nicer list style you ordered :)

### CODE REVIEW

**Comment by yoavweiss** [comment]
@reschke - friendly ping! :)

**Comment by yoavweiss** [comment]
Abandoning as this doesn't seem critical

**Comment by reschke** [comment]
Well, please don't ask for a change if you're not planning to use it...

**Comment by yoavweiss** [comment]
Apologies. Happy to land this if you're willing to review

**Comment by yoavweiss** [comment]
Before:

Screenshot 2020-03-10 at 12 00 46

After:

Screenshot 2020-03-10 at 11 55 23

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -207,12 +207,15 @@ Therefore, features relying on this document to define Client Hint headers MUST
...
 Such features SHOULD take into account the following aspects of the information exposed: 
 
-  - Entropy
-    - Exposing highly granular data can be used to help identify users across multiple requests to different origins. Reducing the set of header field values that can be expressed, or restricting them to an enumerated range where the advertised value is close but is not an exact representation of the current value, can improve privacy and reduce risk of linkability by ensuring that the same value is sent by multiple users.
-  - Sensitivity
-    - The feature SHOULD NOT expose user sensitive information. To that end, information available to the application, but gated behind specific user actions (e.g. a permission prompt or user activation) SHOULD NOT be exposed as a Client Hint.
-  - Change over time
-    - The feature SHOULD NOT expose user information that changes over time, unless the state change itself is also exposed (e.g. through JavaScript callbacks).
+> {: vspace="0"}
+> Entropy:
```
Comment (martinthomson) [inline draft-ietf-httpbis-client-hints.md]: Do you really need the extra indentation?

Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md]: Maybe a better option is to get rid of the nested lists altogether and go with something like:
Screenshot 2020-03-10 at 15 11 55

**Pull request merged**